### ISSUE TITLE
4.1.2 - Move supplementary example to non-normative note

### ISSUE THREAD

**Issue by JasonAment** [w3c/wai-wcag-em#168]
4.1.2 Step 1.2: Define the conformance target

"WCAG 2 Level AA is the generally accepted and recommended target." is non-normative and should be moved to a Note.
